tools/get-or-create-release-branch.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+#
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+#
+
```

Comment:
Nit: Would be good to add a high level explanation of what this is and what it's used for, how to test locally, etc.